docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -0,0 +1,679 @@
+   "metadata": {},
+   "source": [
+    "## Histogramming and binning"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "851054b3-23c7-4b7e-bec4-e3613d87b9d6",
+   "metadata": {},
+   "source": [
+    "We are now ready to bin or histogram our data.\n",
+    "Scipp uses the following terminology:\n",
+    "\n",
+    "- Binning preserves the original data records as a table associated with each bin.\n",
+    "- Histogramming adds up the values from all contributing records into a single value per bin.\n",
+    "\n",
+    "In both cases we need to define bin edges.\n",
+    "As an initial 1-D example, we will compute a wavelength histogram, so we create a variable of wavelength edges:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f73a6379-3369-40a3-aae8-74b6cb026681",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wavelength = sc.linspace(\n",
+    "    'wavelength',\n",
+    "    table.coords['wavelength'].min().value,\n",
+    "    table.coords['wavelength'].max().value,\n",
+    "    num=1001,\n",
+    "    unit='angstrom',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5f6428e-bfb7-44c5-9af1-c4cf9a63a6bf",
+   "metadata": {},
+   "source": [
+    "The table can now be histogrammed using `sc.histogram`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f7a0f173-0224-4932-b34a-e11b85301390",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "histogrammed = sc.histogram(table, bins=wavelength)\n",
+    "histogrammed"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5619166-bc29-4e5c-b486-4e3f651ce625",
+   "metadata": {},
+   "source": [
+    "Alternatively, we can use `sc.bin`, which keeps the underlying events and their metadata:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "469a26d1-7cff-4e14-90a4-faefecb8e3fe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned = sc.bin(table, edges=[wavelength])\n",
+    "binned"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3075af2f-f272-47f5-9683-4c5042bf92eb",
+   "metadata": {},
+   "source": [
+    "Since we used the same bin edges for histogramming and binning, computing the sum of values within each bin (given by `binned.bins.sum()`) yields the same result as histogramming directly.\n",
+    "Therefore only a single line is visible in the following plot:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5ecd8c4f-e0c9-43ae-9a71-13bc6ba54447",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bin_sums = binned.bins.sum()\n",
+    "sc.plot({'histogrammed': histogrammed, 'binned': bin_sums})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bcc22ca-6dff-47ec-9ee8-fb58b59fb644",
+   "metadata": {},
+   "source": [
+    "Since the above combines data from all pixels, the wavelength distribution is not really meaningful.\n",
```

Comment:
Only (experimental) particle-physicists (and related fields) will know what a 'pixel' is in this context. I think this needs more explanation. Could be done as part of the table description above.

Also: we have voxels. Using that term might help because the table contains columns 'voxel_*'.

Reply:
Removed.